### PR TITLE
Add support for using different OCI runtimes

### PR DESCRIPTION
**Pull request by mlaventure**
Required by docker/docker#22983

### CODE REVIEW

**Comment by vdemeester** [comment]
LGTM 🐶
/cc @calavera @MHBauer @vieux @LK4D4 @icecrime

File: types/types.go [hunk 1]
```diff
@@ -252,6 +252,8 @@ type Info struct {
 	ClusterStore       string
 	ClusterAdvertise   string
 	SecurityOptions    []string
+	Runtimes           map[string]Runtime
```
Comment (icecrime) [inline types/types.go]: Should we expose the path in the `/info` output?

Comment (mlaventure) [inline types/types.go]: We do:

```
# docker info
...
Runtimes:
 oci: /usr/local/bin/docker-runc (--debug)
 default: docker-runc
 foo: /usr/local/bin/foo (--debug)
Default Runtime: foo
...
```

It's part of the Runtime struct, we also display the runtime arguments if any were defined

**Comment by calavera** [comment]
LGTM

File: types/types.go (hunk 1 above)
Comment (icecrime) [inline types/types.go]: Sorry my comment was ambiguous: I meant I'm not sure we should expose the path. But I guess it shouldn't hurt, let's do it :+1:

**Pull request merged**